Library/Homebrew/cmd/info.rb

```diff
@@ -62,16 +62,18 @@ def github_fork
   end
 
   def github_info f
-    if f.tap?
-      user, repo = f.tap.split("/", 2)
```

Comment:
I agree. If there was an optimisation here it'd be making it so `f.tap?` and `f.tap` use a shared cache but I think that's overkill on a simple string match.